modular/manager/bucket_migrate_scheduler.go

```diff
@@ -297,6 +311,11 @@ func (s *BucketMigrateScheduler) Init(m *ManageModular) error {
 	}
 	s.executePlanIDMap = make(map[uint64]*BucketMigrateExecutePlan)
 
+	s.bucketCache, err = lru.New(bucketCacheSize)
```

Comment:
What expires in the cache in the LRU, if it has not expired, is there any problem?